### DISCUSSION TITLE
Suggestion: Add to improvements --> Swapping to itwml instead of airportitwml

### DISCUSSION THREAD

**Discussion by jonnijoe1990** [daliansky/XiaoMi-Pro#660] [Ideas]
After some instability problems with the wifi drivers that come with this EFI, I almost thought about buying a wifi card to get native wifi and finally be rid of this.

I managed to get great performance using the "normal" itwml instead of the airport variant. Consider adding this to improvements, if you dont need the airport functionality you are far better of using  itwml + heliport from the OpenIntelWireless section.

"AirportItlwm.kext uses Apple's IO80211Family. It provides certain Airport features but lacks stability compared with itlwm.kext due to the ambiguity of reverse engineering."

Edit: If someone reads this and tries this, be sure to delete the "Force" section in ur config that forces IO80211 to load

**Comment by stevezz3** [comment]
That's a good suggestion. I will write Aiportitlwm -> itlwm related info to README later.

**Comment by jonnijoe1990** [comment]
Sadly I just realized 802.1x is not working with itlwm, making this useless for my purpose since I have to log-in to university wifi alot. I will try to find a fix, but maybe I will have to still buy an airport card. I will update here if I find a solution
